.github/workflows/test.yml

```diff
@@ -26,7 +30,7 @@ jobs:
       run: tox
     - name: "Upload coverage to Codecov"
       if: ${{matrix.python-version == '3.11'}}
```

Comment:
Could you try changing python version to '3.12' here? tox generates the coverage files with the python 3.12 run